src/DialDictionary/include/SplineDialBaseFactory.h

```diff
@@ -14,22 +14,22 @@ class SplineDialBaseFactory {
 
   /// Fill the points starting from a TObject that needs to be pointing to a
   /// graph.  This returns false if it can't get the points.
-  bool FillFromGraph(std::vector<double>& xPoint,
+  static bool FillFromGraph(std::vector<double>& xPoint,
```

Comment:
If you could limit the changes for small stylistic issues for the moment, it would be helpful.  There are some bugs I'm tracking down, and little things like "Class" to "auto" and variable name changes cause conflicts.  Those are easy changes after everything works.

Reply:
Alright, you can let me know when you're done